neuro_flow/expr.py

```diff
@@ -734,6 +740,25 @@ class SimpleOptBoolExpr(BoolExprMixin, Expr[bool]):
     type = bool
 
 
+class EnableExprMixin:
+    @classmethod
+    def convert(cls, arg: Union[AlwaysT, str, bool]) -> Union[bool, AlwaysT]:
+        if isinstance(arg, AlwaysT) or isinstance(arg, bool):
+            return arg
+        if arg == "always()":
```

Comment:
I'm curious why the check is needed

Reply:
This is because of how we process expressions:
1. Parse string to list of TEXT or EXPRESSION
2. Evaluate all items to strings and concat them.
3. Convert that string (expected to be some literal) to python value.

Here is the  code:
```
tokens = list(tokenize(pattern, start=start))
...
self._parsed = PARSER.parse(tokens)
...
for part in self._parsed:
    ...
    val = await part.eval(root)
    ...
    ret.append(str(val))
...
return self.convert("".join(ret))
```
So it's not easy to add something that cannot be represented as python literal. I do the next:
Function `always()` returns instance of `AlwaysT`. Then `AlwaysT.__str__` returns `"always()"`. And finally converter checks for that string and recreates instance of `AlwaysT`.

If there is some easier or cleaner way to achieve the same result, I'll be glad to implement it.

Reply:
Smart but tricky.